specification.md

```diff
@@ -1574,3 +1579,119 @@ to send over the connection. When using a single connection to transmit
 larger messages, TCP buffers can be more efficiently utilized.
 Additionally, it becomes feasible to tune buffer sizes in order to
 optimize the throughput of connections to particular services.
+
+
+
+### Appendix C - examples
+
+This Appendix contains some examples of how the API can be used. Unlike the
+specification language, the examples are in a hypothetical Python binding, which
+is expected to be a relatively frequently used binding.
```

Comment:
Thanks for adding that.

Reply:
I don't think I did. That is, I think it might have been there from the start.

But I still think it's useful to see what you had in mind w.r.t. how an implementation of the spec might look like. Maybe we should open a new issue for that. Might help tell us if we're over or under specifying.

Reply:
Then I missed it on the first read :-)  

> But I still think it's useful to see what you had in mind w.r.t. how an implementation of the spec might look like.

Have a look!  https://github.com/ExaWorks/prototype-saga/tree/feature/radical_prototype/radical